tests/unit_tests/test_FaServerThread.py

```diff
@@ -75,3 +75,72 @@ def test_command_game_host_calls_host_game_invalid_title(connected_socket,
     server_thread.command_game_host(test_game_info_invalid)
     assert mock_lobby_server.games.create_game.mock_calls == []
     server_thread.sendJSON.assert_called_with(dict(command="notice", style="error", text="Non-ascii characters in game name detected."))
+
+# ModVault
+
+# TODO: find a better way
+OneTimeTrue = False
+def one_time_true():
+    global OneTimeTrue
+    if OneTimeTrue:
+        return False
+    OneTimeTrue = True
+    return True
+
+@mock.patch('src.lobbyconnection.QSqlQuery')
+@mock.patch('src.lobbyconnection.Config')
+def test_mod_vault_start(mock_config, mock_query, connected_socket, mock_lobby_server):
+    server_thread = LobbyConnection(connected_socket, mock_lobby_server)
+    server_thread.sendJSON = mock.Mock()
+    mock_query.return_value.size.return_value = 1
+    mock_query.return_value.next = one_time_true
+    server_thread.command_modvault({'type': 'start'})
+    server_thread.sendJSON.assert_called_once()
+    # call, method:attributes, attribute_index
+    assert server_thread.sendJSON.mock_calls[0][1][0]['command'] == 'modvault_info'
```

Comment:
Indexing into mock_calls like this is pretty voodoo, even if you state the parameters just above.